homeassistant/components/utility_meter/__init__.py

```diff
@@ -38,13 +39,14 @@
 ATTR_TARIFFS = "tariffs"
 
 DEFAULT_OFFSET = timedelta(hours=0)
+DEFAULT_METER_TYPE = MANUAL
 
 
 METER_CONFIG_SCHEMA = vol.Schema(
     {
         vol.Required(CONF_SOURCE_SENSOR): cv.entity_id,
         vol.Optional(CONF_NAME): cv.string,
-        vol.Optional(CONF_METER_TYPE): vol.In(METER_TYPES),
+        vol.Optional(CONF_METER_TYPE, default=DEFAULT_METER_TYPE): vol.In(METER_TYPES),
```

Comment:
I just realised something very important 😄 

The CONF_METER_TYPE is optional that means you **already** don't need to define the meter type

That is, your purpose (to not reset the utility_meter) already exists and is working!

Reply:
What is missing is in the documentation, in there it should state (as seen in your PR) that the cycle is **optional**

Reply:
Yes, I also noticed that the CONF_METER_TYPE was already programmed as optional but not default value was assigned. In this way I feel it is better, because it eliminates ambiguity by defaulting to a given cycle.
Also added some extra clarification in manual text. In the `{% configuration %}` section of the docs, the `cycle`was already labeled as `required: false`

Reply:
When it is `Optional` and no `default` is assigned, it gets assigned `None`